run_segmentation.py

```diff
@@ -74,10 +74,12 @@ def main(method_, raw_dir_, supp_dir_, val_dir_, config_):
     sep = np.linspace(0, len(segment_sites), gpus + 1).astype(int)
 
     processes = []
-    for i, gpu in enumerate(gpus):
+    # for i, gpu in enumerate(gpus):
+    for i in range(gpus):
         _sites = segment_sites[sep[i]:sep[i + 1]]
```

Comment:
This change would make the processes always run with the order on gpu 0, 1, 2..., and would error out if gpu 0 doesn't have enough memory. You need to check available gpu memory and only run on available gpus.

Reply:
Ok i have fixed this

Reply:
The behavior still looks the same to me. Maybe I have missed something. Let's chat about this